dietpi/login

```diff
@@ -189,7 +189,7 @@
 				fi
 
 				# - Prompt and wait if this script runs in other session already
-				if (( $(pgrep 'login' 2> /dev/null | wc -l) >= 2 )); then
+				if (( $(pgrep -c 'login') >= 2 )); then
```

Comment:
@Fourdee 
Quick check required if `-c` is available on Jessie as well, e.g. `-i` is not.